classes/PodsData.php

```diff
@@ -2196,11 +2196,13 @@ public static function get_table_columns ( $table ) {
      * @since 2.0
      */
     public static function get_column_data ( $column_name, $table ) {
-        $describe_data = mysql_query( 'DESCRIBE ' . $table );
+	    global $wpdb;
+
+        $describe_data = $wpdb->get_results( 'DESCRIBE ' . $table, ARRAY_A );
 
         $column_data = array();
 
-        while ( $column_row = mysql_fetch_assoc( $describe_data ) ) {
+        foreach ( $describe_data as $column_row ) {
             $column_data[] = $column_row;
         }
 
```

Comment:
This function could be simplified by just using `$describe_data` instead of creating a new var (`$column_data`) since the contents will be the same when using `$wpdb`.